From e680be170761fdbfb3c9bc5ee51a5f6002e586ca Mon Sep 17 00:00:00 2001
From: Dmitry Timoshkov <dmitry@baikal.ru>
Date: Wed, 7 Dec 2016 16:10:31 +0800
Subject: user32: Don't force a combobox repaint on WM_SIZE.

This breaks z-order based painting and causes side effects for
applications that during the WM_PAINT processing reference internal
data associated with a not fully initialized window.
---
 dlls/user32/combo.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/user32/combo.c b/dlls/user32/combo.c
index a2e8b3c..501bfe8 100644
--- a/dlls/user32/combo.c
+++ b/dlls/user32/combo.c
@@ -1558,7 +1558,7 @@ static void COMBO_Size( LPHEADCOMBO lphc )
 		  &lphc->buttonRect,
 		  &lphc->droppedRect);
 
-  CBResetPos( lphc, &lphc->textRect, &lphc->droppedRect, TRUE );
+  CBResetPos( lphc, &lphc->textRect, &lphc->droppedRect, FALSE );
 }
 
 
-- 
2.9.0

